src/GitHub.InlineReviews/ViewModels/CommentViewModel.cs

```diff
@@ -130,22 +131,41 @@ protected void AddErrorHandler<T>(ReactiveCommand<T> command)
 
         async Task DoDelete(object unused)
         {
-            try
-            {
-                ErrorMessage = null;
-                IsSubmitting = true;
+            string deleteCommentMessageBoxText = "Are you sure you want to delete this comment?";
+            string deleteCommentCaption = "Delete Comment";
+            MessageBoxButton deleteCommentButton = MessageBoxButton.YesNo;
+            MessageBoxImage deleteCommentImage = MessageBoxImage.Question;
 
-                await Thread.DeleteComment.ExecuteAsyncTask(new Tuple<int, int>(PullRequestId, DatabaseId));
-            }
-            catch (Exception e)
-            {
-                var message = e.Message;
-                ErrorMessage = message;
-                log.Error(e, "Error Deleting comment");
-            }
-            finally
+            MessageBoxResult result = MessageBox.Show(deleteCommentMessageBoxText, deleteCommentCaption, deleteCommentButton, deleteCommentImage);
```

Comment:
I was searching for something like this in the codebase that I could use as an example

Reply:
Adding another comment.

Reply:
Yeah, there's also `VsShellUtilities.ShowMessageBox` - I'm not sure if we should be using that one or this one? @AArnott would you know?